src/main/java/com/sweepy/converter/randomConverter.java

```diff
@@ -0,0 +1,30 @@
+package com.sweepy.converter;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+public class randomConverter {
+
+    Map<Integer, String> map = null;
+    Random r = null;
+    public randomConverter() {
+        this.map= new HashMap<>();
+        this.r = new Random();
+    }
+    public String encode(String longUrl) {
+
+
+        int key = r.nextInt(Integer.MAX_VALUE);
+
+        while (map.containsKey(key)) {
+            key = r.nextInt(Integer.MAX_VALUE);
+        }
+        map.put(key, longUrl);
+        return "" + key;
+    }
+
+    public String decode(String shortUrl) {
+        return map.get(Integer.parseInt(shortUrl.replace("http://www.sweepy.com/", "")));
+    }
```

Comment:
should use the database to replace the In-Memory map

Reply:
I didn't update github but this method is removed therefore fixed.